src/vue/mixins/sales.mixin.js

```diff
@@ -0,0 +1,105 @@
+import config from '@/config'
```

Comment:
Create a single component instead of this mixin with some prop, something like `isUserSales` and use it for `all` and `user owned` sales